### DISCUSSION TITLE
Simple landing page that must be able to handle sudden spike in traffic (50k users). Can Vercel handle that?

### DISCUSSION THREAD

**Discussion by williamlmao** [vercel/vercel#7675] [Help]
I have a simple website for an event where we are expecting 50k users all around the same 1 hour window. The site cannot crash during this high traffic time. 

Since the fair use policy does not allow load testing, how can I be sure that Vercel can handle this? I'd be upgrading to the pro plan if I went with Vercel. Thanks!

**Comment by domeccleston** [comment]
Hi there! We'd be happy to help you with this. Will you drop me an email on dom.eccleston@vercel.com so I can chat further about your launch and get you any resources you need?
